src/world/time_server.cpp

```diff
@@ -30,82 +30,105 @@
 #include "daily_tally.h"
 #include "world_server.h"
 
-int32 time_server(time_point tick, CTaskManager::CTask* PTask)
+int32 time_server(timer::time_point tick, CTaskManager::CTask* PTask)
 {
     TracyZoneScoped;
+    // Track elapsed ticks.
+    static auto tickNum = 0;
+    ++tickNum;
 
-    TIMETYPE     VanadielTOTD = CVanaTime::getInstance()->SyncTime();
-    WorldServer* worldServer  = std::any_cast<WorldServer*>(PTask->m_data);
+    WorldServer* worldServer = std::any_cast<WorldServer*>(PTask->m_data);
 
-    // Weekly update for conquest (sunday at midnight)
-    static time_point lastConquestTally  = tick - 1h;
-    static time_point lastConquestUpdate = tick - 1h;
-    if (CVanaTime::getInstance()->getJstWeekDay() == 1 && CVanaTime::getInstance()->getJstHour() == 0 && CVanaTime::getInstance()->getJstMinute() == 0)
+    /**
+     * Earth-based ticks.
+     * Uses the JST equivalent of the current timer tick. (steady_clock -> system_clock)
+     */
+
+    // Earth time points
+    earth_time::time_point jstTime    = earth_time::time_point(timer::to_utc(tick));
+    auto                   jstHour    = earth_time::jst::get_hour(jstTime);
+    auto                   jstWeekday = earth_time::jst::get_weekday(jstTime);
+
+    // Static variable for the next tick
+    static earth_time::time_point nextHourlyTick = std::chrono::ceil<std::chrono::hours>(jstTime);
```

Comment:
`static const auto`

Reply:
This is updated during ticks so went with `static auto`.